test/actor_test.py

```diff
@@ -369,15 +369,7 @@ def getpid(self):
             # If we can successfully create an actor, that means that enough
             # GPU resources are available.
             a = Actor.remote()
-            pid = ray.get(a.getpid.remote())
-
-            # Make sure that we can't create another actor.
-            with self.assertRaises(Exception):
```

Comment:
Should we propagate an error if we can't find enough resources to create an actor?

Reply:
I think no because other actors may go out of scope very soon enabling this actor to be created.

Similarly, a task may not be able to be run right away, but soon some resources will free up and then it can run.